src/test/ui-fulldeps/issue-11881.rs

```diff
@@ -1,52 +0,0 @@
-// run-pass
```

Comment:
Not sure what the exact original bug was so I don't know how to rewrite it to not depend on rustc_serialize.